### DISCUSSION TITLE
启用v2ray代理后firefox总是非常容易卡死

### DISCUSSION THREAD

**Discussion by Geek-Joe** [v2fly/v2ray-core#1151] [Q&A] [answered]
1. 不走代理完全正常，走代理特别容易整个浏览器都卡死（尤其是看完一个youtube视频准备切换下一个时），等几秒或几几十秒后大概率能自己恢复（晚上更容易出现）。
2. 规则我已经感觉简化得不能再简化了，麻烦大佬们看看问题出在哪儿，感激不尽（v2ray版本：v4.40.1 Firefox:89.0.2）
3. 配置文件 config.txt、access.log 和 error.log 文件请到网盘下载(时间：23时53分内) https://emma.cloud.tabdigital.eu/s/rdM7qHet674zYf4
4. 环境：Fedora 34,firefox 90,v2ray v4.40.1

**Comment by brandon3343** [comment]
那就不是卡死，是你的服务器连接速度太慢了，然后这些时间是等待的

**Reply by Geek-Joe** [reply]
整个浏览器都动不了，所有UI操作都无法进行，不是单纯的网页打开慢。这个V2ex上也有很多人有类似的反馈，感觉开了v2ray卡顿严重，而firefox一直都是以前的版本。
v2ex帖子连接：https://www.v2ex.com/t/790400
关掉代理就好了，感觉有些相关性，目前并无明确证据，但是从v2ex上看有类似遭遇的人不少

**Reply by brandon3343** [reply]
v2ex搜了一下，好像没搜到浏览器卡顿的......不然你试试把ipv6也关了看看，另外卡顿的时候，你看下CPU情况呀

**Reply by Geek-Joe** [reply]
观察了一段时间，可能还是Firefox自身的问题。

1. 出问题的时候Firefox的CPU的占用明显异常升高，某些内核被长时间占满（虽然整体只占了24%左右）
2. 换Chrome却没有任何问题

**Reply by georgh-** [reply]
Using Linux here, I can reproduce the same behavior. It started few months ago, I think it's purely Firefox-related, as other people also commented proxy issues using "FoxyProxy" extension (it is also reproduced without).

Firefox will use 100% CPU of the core it's assigned to (in Windows it's harder to see CPU usage as it's reported as a percentage of all cores/CPUs).

Using Chromium, no issues whatsoever.

**Reply by Geek-Joe** [reply]
> Using Linux here, I can reproduce the same behavior. It started few months ago, I think it's purely Firefox-related, as other people also commented proxy issues using "FoxyProxy" extension (it is also reproduced without).
> 
> Firefox will use 100% CPU of the core it's assigned to (in Windows it's harder to see CPU usage as it's reported as a percentage of all cores/CPUs).
> 
> Using Chromium, no issues whatsoever.

This problem has been resolved on Firefox 93.0

**Comment by Geek-Joe** [comment, marked as answer]
经实测，Firefox 93.0 版本不再有该问题！
